x-pack/platform/plugins/shared/fleet/server/routes/remote_synced_integrations/index.ts

```diff
@@ -49,4 +54,38 @@ export const registerRoutes = (router: FleetAuthzRouter) => {
       },
       getRemoteSyncedIntegrationsStatusHandler
     );
+
+  router.versioned
+    .get({
+      path: REMOTE_SYNCED_INTEGRATIONS_API_ROUTES.INFO_PATTERN,
+      security: {
+        authz: {
+          requiredPrivileges: [
+            FLEET_API_PRIVILEGES.SETTINGS.READ,
+            FLEET_API_PRIVILEGES.INTEGRATIONS.READ,
```

Comment:
I think the flyout has to be adapted with the correct minimal privileges: https://github.com/elastic/kibana/blob/a616a40f9307cd34c6a28c7192cfee146ca5ad76/x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/output_form_remote_es.tsx#L391-L404

We should test that the API works correctly with a user role having only minimal privileges